rules.py

```diff
@@ -422,11 +423,11 @@ def _get_ocd_data(self):
                     if last_mod_date < latest_github_commit_date:
                         self._download_data(countries_file)
         ocd_id_codes = set()
-        with io.open(countries_file, mode="rb") as fd:
-            for line in fd:
-                if line is not "":
-                    # TODO use a CSV Reader
-                    ocd_id_codes.add(line.split(b",")[0])  
+        with open(countries_file) as csvfile:
+            csv_reader = csv.DictReader(csvfile)
+            for row in csv_reader:
+                if row['id']:
```

Comment:
Should be

```
if 'id' in row and row['id']:
```

Also, have we confirmed that this change works with the existing files?

Reply:
Tested this change with all the EU feeds.